presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -892,11 +892,55 @@ var QueryDetail = React.createClass({
     },
     renderFailureInfo: function() {
         var query = this.state.query;
+        var taskRow = "";
```

Comment:
This level of complexity is not required. We're ok with null column values - they simply don't get displayed. See the other fields - we only construct specific templates when they're absolutely necessary.